File: 1614-maximum-nesting-depth-of-the-parentheses/1614-maximum-nesting-depth-of-the-parentheses.java
class Solution {
    public int maxDepth(String s) {
        int count = 0, ans = 0;
        for(char c : s.toCharArray()) {
           if(c == '(') count++;
           else if(c == ')') count--;
           ans = Math.max(ans, count);
        }

        return ans;
    }
}

File: 1614-maximum-nesting-depth-of-the-parentheses/1614-maximum-nesting-depth-of-the-parentheses.cpp
class Solution {
public:
    int maxDepth(string s) {
        int n = s.size(), count = 0, ans = 0;
        for(int i = 0; i<n; i++) {
            if(s[i] == '(') count++;
            else if(s[i] == ')') count--;
            ans = max(ans, count);
        }

        return ans;
    }
};